### ISSUE TITLE
Walk with fastai revisited - Project

### ISSUE THREAD

**Issue by manisnesan** [manisnesan/fastchai#34]
Label: Current

The goal is to create a hf space showcasing a deep learning application as part of [Walk with fastai course](https://store.walkwithfastai.com/walk-with-fastai-revisited) project. Personal goal is to get myself comfortable with the MidLevel API. 

**Project Ideas**
- [Finalized] FashionMNIST - use only low level fastai api
- ~Spot the mask - use only low level fastai api~

**Tasks**
- [x] Download & Load the [fashion_mnist](https://huggingface.co/datasets/fashion_mnist) dataset using HuggingFace Dataset.
- [x] Create fastai dataloaders from hf dataset using [Mid Level API](https://docs.fast.ai/tutorial.imagenette.html) such as Datasets from [DataCore](https://docs.fast.ai/data.core.html).
- [x] Train a model and Predict items
- [x] Add gradio interface and Deploy to HF space
- [x] Add a blog post

**Comment by manisnesan** [comment]
[Code - Google Colab](https://colab.research.google.com/drive/1dOG80UWGvZiXTNJDnkBPZq7uDNkHo-TT?usp=sharing)

**Comment by manisnesan** [comment]
Gradio Space for Fashion MNIST Classifier - https://huggingface.co/spaces/msivanes/fashion-mnist-classifier

**Comment by manisnesan** [comment]
Idea

From fastbook Chapter 11 - Data Munging with fastai's Mid-Level API
> What can we do when the data block API is not flexible enough to accommodate our particular use case? . 
> For this, we need to use fastai's mid-level API for processing data. The data block API is built on top of that layer, so it will allow you to do everything the data block API does, and much much more.
> It encompasses Transforms, Pipeline, TfmdLists, Datasets, Callbacks and General Optimizer

Context
- Dataset: Fashion MNIST classification dataset containing black white images of size 28x28 of apparels along with the label such as t-shirt, ankle boot containing 10 classes. 
- Goal is to create a classifier predicting the target class given an black white image

Problem
- 

Outline
- Idea
- Context
- Problem
- Code Walkthrough of the Classifier application
- Troubleshooting
- Takeaways

**Comment by manisnesan** [comment]
Published [post](https://manisnesan.github.io/chrestotes/posts/2023-04-14-fashion-mnist.html)

Minor Rendering issues

- yaml showing up

toc: true
badges: true
comments: true
categories: [image-classification, machine-learning]
image: images/chart-preview.png


- Change to bulleted 
> Steps involved to prepare the data

- Change to bulleted 

> Steps involved to prepare the data
